packages/bruno-app/src/components/Modal/index.js

```diff
@@ -68,10 +71,15 @@ const Modal = ({
   closeModalFadeTimeout = 500
 }) => {
   const [isClosing, setIsClosing] = useState(false);
-  const escFunction = (event) => {
-    const escKeyCode = 27;
-    if (event.keyCode === escKeyCode) {
-      closeModal({ type: 'esc' });
+
+  const handleKeydown = ({ keyCode }) => {
+    switch (keyCode) {
+      case ESC_KEY_CODE: {
+        return closeModal({ type: 'esc' });
+      }
+      case ENTER_KEY_CODE: {
+        return handleConfirm();
```

Comment:
Not every instance of Modal component has the handleConfirm behavior defined.

Reply:
I have addressed this @sanjai0py !